components/create/CreateCollection.vue

```diff
@@ -266,37 +266,4 @@ onMounted(() => {
 })
 </script>
 
-<style lang="scss" scoped>
-@import '@/styles/abstracts/variables';
-
-.o-field:not(:last-child) {
-  margin-bottom: 2rem;
-}
-
-.column {
-  max-width: 36rem;
-  padding: 4rem;
-
-  @include desktop() {
-    @include ktheme() {
-      background-color: theme('background-color');
-      box-shadow: theme('primary-shadow');
-    }
-  }
-
-  @include touch() {
-    padding: 0 1rem;
-    box-shadow: none !important;
-  }
-}
-
-@include desktop() {
-  .columns {
-    padding: 5.25rem 0;
-
-    @include ktheme() {
-      background-color: theme('k-primaryLight');
-    }
-  }
-}
-</style>
+<style lang="scss" scoped src="./create.scss"></style>
```

Comment:
hmmm 🤔

Reply:
oh btw, this is similar with `@import './create.scss'` on scss

Reply:
hmmm.. weird new pattern but ok. I would have preferred that you put component scss in the right folder

Reply:
updated, move it to `styles` folder